lib/puppet/provider/iis_site/webadministration.rb

```diff
@@ -49,10 +49,20 @@ def update
 
     cmd << self.class.ps_script_content('serviceautostartprovider', @resource)
 
+    if @resource[:authenticationinfo]
+      @resource[:authenticationinfo].each do |auth,enable|
+        args = Array.new
+        args << "-Filter 'system.webserver/security/authentication/#{auth}Authentication'"
+        args << "-PSPath 'IIS:\\'"
+        args << "-Location '#{@resource[:name]}'"
+        args << "-Name enabled"
+        args << "-Value #{@resource[:authenticationinfo][auth]}"
+        cmd << "Set-WebConfigurationProperty #{args.join(' ')} -ErrorAction Stop"
+      end
+    end
 
-
-    inst_cmd = cmd.join
-
+    inst_cmd = cmd.join("\n")
```

Comment:
Interested as to why this was changed.  Instead should you just add a "\n" on line 60?

Reply:
Fine